core/uemis-downloader.c

```diff
@@ -73,6 +73,7 @@ static int mbuf_size = 0;
 static int max_mem_used = -1;
 static int next_table_index = 0;
 static int dive_to_read = 0;
+static uint32_t mindiveid = 0xFFFFFFFF;
```

Comment:
given `uemis_init()` sets the same value near the function entry the above initialization isn't really needed.